riscv/execute.cc

```diff
@@ -173,6 +175,13 @@ static inline reg_t execute_insn_logged(processor_t* p, reg_t pc, insn_fetch_t f
 
   try {
     npc = fetch.func(p, fetch.insn, pc);
+
+    if (p->get_log_commits_enabled()) {
```

Comment:
Can you add an additional flag `--trace`, for example, instead of overloading `log-commits`.